OpenOrchestrator/scheduler/runner.py

```diff
@@ -142,6 +143,24 @@ def clone_git_repo(repo_url: str) -> str:
     return repo_path
 
 
+def clear_repo_folder() -> None:
+    """Completely remove the repos folder."""
+    repo_folder = get_repo_folder_path()
+    subprocess.run(['rmdir', '/s', '/q', repo_folder], check=False, shell=True)
+
+
+
+def get_repo_folder_path() -> str:
+    """Gets the path to the folder where robot repos should be saved.
+
+    Returns:
+        str: The absolute path to the repo folder.
+    """
+    desktop_path = os.path.expanduser("~\\Desktop")
```

Comment:
Use `os.path.join` to build the path:

```suggestion
    desktop_path = os.path.expanduser(os.path.join(`~`, 'Desktop'))
```